core/src/components/input/input.ionic.scss

```diff
@@ -15,12 +18,26 @@
   min-height: 48px;
 }
 
+// Input Bottom Content
+// ----------------------------------------------------------------
+
+.input-bottom {
+  @include padding-horizontal(0, 0);
+}
+
 // Input Hint Text
 // ----------------------------------------------------------------
 
 .input-bottom .helper-text {
   // TODO(FW-6112): Verify the design token is correct once it's available and remove the hardcoded value.
   color: var(--ionic-color-neutral-600, #535353);
+  // TODO(FW-6112): Verify the design token is correct once it's available and remove the hardcoded value.
+  font-weight: var(--ionic-font-medium, 500);
+}
```

Comment:
While I was implementing the valid state, I noticed that some styles were missing from the helper text.